app/assets/stylesheets/_users.scss

```diff
@@ -212,6 +212,14 @@ form {
   margin-top: 3em;
 }
 
+.forgot-password {
+  #show_password:checked + [for="show_password"]::before {
+    -webkit-transform: rotate(45deg);
```

Comment:
Properties should be ordered -moz-transform, -ms-transform, -o-transform, -webkit-transform